src/js/background/background.js

```diff
@@ -109,21 +109,29 @@ async function storeRuntimeData() {
   if (!relayApiSource) {
     return;
   }
-  const currentPremiumAvailabilityResponse = await fetch(
+  const runtimeDataResponse = await fetch(
     `${relayApiSource}/runtime_data`,
     {
       headers: { Accept: "application/json" },
     },
   );
-  const currentPremiumAvailability = await currentPremiumAvailabilityResponse.json();
-
+  const runtimeData = await runtimeDataResponse.json();
+  
   browser.storage.local.set({
     premiumCountries: {
-      PREMIUM_PLANS: currentPremiumAvailability.PREMIUM_PLANS,
+      PREMIUM_PLANS: runtimeData.PREMIUM_PLANS,
+      fetchedAt: Date.now(),
+    },
+    waffleFlags: {
+      WAFFLE_FLAGS: runtimeData.WAFFLE_FLAGS,
+      fetchedAt: Date.now(),
+    },
+    bundlePlans: {
+      BUNDLE_PLANS: runtimeData.BUNDLE_PLANS,
       fetchedAt: Date.now(),
     },
-    introPricingEndDate: {
-      INTRO_PRICING_END: currentPremiumAvailability.INTRO_PRICING_END,
+    phonePlans: {
+      PHONE_PLANS: runtimeData.PHONE_PLANS,
       fetchedAt: Date.now(),
```

Comment:
Might as well include the new (monthly and yearly) Premium plans:

```suggestion
    periodicalPremiumPlans: {
      PERIODICAL_PREMIUM_PLANS: runtimeData.PERIODICAL_PREMIUM_PLANS,
      fetchedAt: Date.now(),
    },
    phonePlans: {
      PHONE_PLANS: runtimeData.PHONE_PLANS,
      fetchedAt: Date.now(),
```

Reply:
done